app/tests/e2e/issuer/test_issuer_ld_ed25519.py

```diff
@@ -0,0 +1,424 @@
+import pytest
+from aries_cloudcontroller import (
+    AcaPyClient,
+    Credential,
+    LDProofVCDetail,
+    LDProofVCDetailOptions,
+)
+from assertpy import assert_that
+from fastapi import HTTPException
+
+from app.models.issuer import SendCredential
+from app.routes.issuer import router as issuer_router
+from app.routes.oob import router as oob_router
+from app.routes.wallet import router as wallet_router
+from app.tests.util.ecosystem_connections import FaberAliceConnect
+from app.tests.util.trust_registry import DidKey
+from app.tests.util.webhooks import check_webhook_state
+from shared import RichAsyncClient
+
+CREDENTIALS_BASE_PATH = issuer_router.prefix
+OOB_BASE_PATH = oob_router.prefix
+WALLET = wallet_router.prefix
+
+# This is the json of the below credential
+# {
+#     "type": "ld_proof",
+#     "connection_id": "",
+#     "protocol_version": "v2",
+#     "ld_credential_detail": {
+#         "credential": {
+#             "@context": [
+#                 "https://www.w3.org/2018/credentials/v1",
+#                 "https://www.w3.org/2018/credentials/examples/v1",
+#             ],
+#             "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+#             "credentialSubject": {
+#                 "degree": {
+#                     "type": "BachelorDegree",
+#                     "name": "Bachelor of Science and Arts",
+#                 },
+#                 "college": "Faber College",
+#             },
+#             "issuanceDate": "2021-04-12",
+#             "issuer": "",
+#         },
+#         "options": "",
+#     },
+# }
+
+credential = SendCredential(
+    type="ld_proof",
+    connection_id="",
+    protocol_version="v2",
+    ld_credential_detail=LDProofVCDetail(
+        credential=Credential(
+            context=[
+                "https://www.w3.org/2018/credentials/v1",
+                "https://www.w3.org/2018/credentials/examples/v1",
+            ],
+            type=["VerifiableCredential", "UniversityDegreeCredential"],
+            credentialSubject={
+                "degree": {
+                    "type": "BachelorDegree",
+                    "name": "Bachelor of Science and Arts",
+                },
+                "college": "Faber College",
+            },
+            issuanceDate="2021-04-12",
+            issuer="",
+        ),
+        options=LDProofVCDetailOptions(proofType=""),
```

Comment:
All credentials in this module will have `"proofType": "Ed25519Signature2018"`. Note to update the commented example of json too